azext_iot/iothub/params.py

```diff
@@ -45,3 +50,189 @@ def load_iothub_arguments(self, _):
             help="Maximum interval of time, in seconds, that the digital twin command will wait for the result.",
             arg_group="Timeout"
         )
+
```

Comment:
Assuming these additions are mostly copied over from prior locations.

Reply:
Except for 
![image](https://user-images.githubusercontent.com/28658112/174679080-32055ce9-03c9-44cf-b6c2-9e9edecbb984.png)

Reply:
yup